bin/localstack/start_localstack

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+docker-compose -f infra/localstack.yaml up -d
+
+./bin/localstack/wait_localstack_running
```

Comment:
Suggestion (blocking): Would you inline all the waiting scripts here? I am not sure how much value they provide as separate scripts.